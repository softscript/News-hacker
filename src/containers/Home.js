import React, { Component } from 'react';
import {  Link } from 'react-router-dom';

class Home extends Component {
    render() {
        return (
            <h1>Home</h1>
        )
    }
}

export default Home