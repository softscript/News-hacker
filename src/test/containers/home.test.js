import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});
import Home from './../../containers/Home';
const title = 'Home';
let wrapped = shallow(<Home>{title}</Home>);
describe('Title', () => {
  it('should render the Title Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Titles children', () => { 
    expect(wrapped.find('h1').text()).toEqual(title);
  });
});