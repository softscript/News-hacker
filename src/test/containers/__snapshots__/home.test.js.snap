// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the Title Component correctly 1`] = `
<h1>
  Home
</h1>
`;
